packages/components/src/components/heatmap/HeatMap.tsx

```diff
@@ -29,23 +29,29 @@ import { naturalSort } from '../../util/utils';
 
 export interface HeatMapProps {
     schemaQuery: SchemaQuery
+    displayNamePath?: Array<string> // path within a query row to use for displaying the y axis and by which to sort the rows of data
```

Comment:
Might be worth adding an example to the end of this comment. I assume you mean "path" as in set of key's for the response object, ["Something", "value"]?

Reply:
Ah, I see the "example" in the default props.